packages/errors/src/errors.ts

```diff
@@ -188,6 +188,8 @@ export const AllCypressErrors = {
 
         ${fmt.highlightSecondary(arg1.response)}
 
+        Because you passed the ${fmt.flag(`--record`)} flag, this run cannot proceed because it requires a valid response from our servers.
```

Comment:
Just removing the double 'because'

```suggestion
        Because you passed the ${fmt.flag(`--record`)} flag, this run cannot proceed since it requires a valid response from our servers.
```